### PR TITLE
Fix/refactorbugs

### PR DESCRIPTION
**Pull request by Kanderson58**
# Description

- Bug fix CSS errors
- Bug fix autofocus
- Use GraphQL to mutate housename

## Type Of Change

- [x]  Bug fix
- [ ]  New feature
- [x]  Refactoring
- [ ]  Testing

# How Has This Been Tested?

- [x]  local host
- [x]  dev tools
- [ ]  cypress

# Related Issues:

- [ ]  closes 

# Checklist:

- [x]  My code follows the Turing's guidelines of this project
- [x]  I have performed a self-review of my own code
- [x]  No console error messages

### CODE REVIEW

File: src/Components/ChoreCard/ChoreCard.js
Comment (LeftyLincoln) [inline src/Components/ChoreCard/ChoreCard.js:1]: Great work adding the tabbing for our chore cards, nice catch

File: src/Components/Dashboard/Dashboard.js
Comment (LeftyLincoln) [inline src/Components/Dashboard/Dashboard.js:1]: Love this fixing our autofocus!

**Review by LeftyLincoln — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -16,11 +16,11 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "cypress": "cypress open"
+    "cypress": "cypress open",
+    "start": "GENERATE_SOURCEMAP=false react-scripts start"
```
Comment (qrispi) [inline package.json:23]: Hhhmmmmm.....

File: src/Components/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -46,13 +46,12 @@ const Dashboard = ({ email, setId }) => {
   });
 
   const weekButtons = daysOfWeek.map((day) => {
-    if (day === "monday") {
+    if (day === dayOfWeek) {
       return (
         <button
           key={day}
-          className={`day-btn ${day}`}
+          className={`day-btn ${day} selected`}
```
Comment (qrispi) [inline src/Components/Dashboard/Dashboard.js:53]: oooo love the conditional CSS classes

**Review by qrispi — APPROVED** [review]
LGTM! Love how clean our code is getting!

**Pull request merged**